cli/crates/cli/Cargo.toml

```diff
@@ -11,14 +11,14 @@ repository = "https://github.com/grafbase/grafbase"
 categories = ["command-line-utilities"]
 
 [dependencies]
-backtrace = "0.3"
+backtrace = "0.3.66"
```

Comment:
Why?

Reply:
Merge error I think, it's been fixed on the other PR

Reply:
Fixed